go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -152,11 +142,9 @@ func (npw *nodePortWatcher) updateServiceFlowCache(service *kapi.Service, add bo
 					npw.ofm.updateFlowCacheEntry(key, nodeportFlows)
 				} else {
 					npw.ofm.updateFlowCacheEntry(key, []string{
-						fmt.Sprintf("cookie=%s, priority=110, in_port=%s, %s, tp_dst=%d, "+
-							"actions=%s",
-							cookie, npw.ofportPhys, flowProtocol, svcPort.NodePort, actions),
-						fmt.Sprintf("cookie=%s, priority=110, in_port=%s, %s, tp_src=%d, "+
-							"actions=output:%s",
+						fmt.Sprintf("cookie=%s, priority=100, in_port=%s, %s, tp_dst=%d, actions=%s",
```

Comment:
nodeport related flows have priority 110, why was this changed to 100 ?

Reply:
this PR bumped node port related flows priority to 110 

https://github.com/ovn-org/ovn-kubernetes/commit/2c0ec2337e590e2287c57f80975dadb79a9ad1cf

Reply:
I mistakenly changed this when reverting the commit. fixed.